<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Studio</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="select_ctrls">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Select Ctrls From Node</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="show_box">
           <item>
            <property name="text">
             <string>Sandbox</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AnythingGoes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SP</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="paths_box">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load_anim">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Load Anim</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Done</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="prev_key">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;&lt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="GlobalTester">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Global_Ctrl Movements</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="next_key">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&gt;&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="prev_key_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;&lt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Rotate90">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Rotate 90 Sel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="next_key_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&gt;&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>WIP</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QPushButton" name="copy_anim">
                 <property name="minimumSize">
                  <size>
                   <width>25</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Copy Anim</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="paste_anim">
                 <property name="minimumSize">
                  <size>
                   <width>25</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Paste Anim</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="save_anim_to_desktop">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>Save Anim to Dekstop</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="load_anim_from_desktop">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>Load Anim from Dekstop</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="clear">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(239, 41, 41)</string>
         </property>
         <property name="text">
          <string>Clear Keys</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Mixamo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mixamo_mocaps">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load_mocap">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Load Mocap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="restore_mutant_rig">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Restore Mutant Rig</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Calisthenics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="create_rom">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clean_rom">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Clean</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="combo_shows_combo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="search_bar_line">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_assets_combo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="run_import">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
